axelor-production/src/main/resources/domains/ProductionConfig.xml

```diff
@@ -25,13 +25,19 @@
     <many-to-one name="partFinishMoMessageTemplate" title="Message template"
       ref="com.axelor.message.db.Template"/>
 
+    <integer name="capacity" title="Capacity" selection="production.config.capacity.select"
+      default="1"/>
+
 
     <finder-method name="findByCompany" using="company"/>
 
     <extra-code>
 		<![CDATA[
    			public static final int REALIZE_START = 0;
    			public static final int REALIZE_FINISH = 1;
+
+   			public static final int PRODUCTION_CAPACITY_INFINITE = 1;
+   			public static final int PRODUCTION_CAPACITY_FINITE = 2;
```

Comment:
Please rename these variables to `INFINITE_CAPACITY_SCHEDULING`` and `FINITE_CAPACITY_SCHEDULING`